packages/core/lib/create.js

```diff
@@ -124,7 +119,12 @@ var Create = {
       );
     }
 
-    copy.file(from, to, callback);
+    try {
+      await copy.file(from, to);
+      callback();
+    } catch (error) {
+      callback(error);
```

Comment:
```suggestion
      return callback(error);
```

I think this needs to be returned.